README.md

```diff
@@ -2154,7 +2154,18 @@ An open question. Answer based on your real experience. You can highlight one or
   * man
   * nslookup or dig
   * pwd
-  * df</summary><br><b>
+  * df
+  * file
+  * cat
+  * less
+  * history
+  * clear
+  * cp
+  * mv
+  * mkdir
+  * find
+  * alias
+  * exit</summary><br><b>
```

Comment:
can you remove "clear" and "exit" from the list? the reason is that they are not that interesting or important to learn (most folks will just use shortcuts to exit or clear the screen, not the actual commands). Let's also remove "less" and "file".